src/utils/uint256.ts

```diff
@@ -10,6 +10,14 @@ export { UINT_128_MAX, UINT_256_MAX };
 /**
  * Convert Uint256 to bigint
  * Legacy support Export
+ * @param uint256 Uint256 value to convert to bigint
+ * @returns {bigint} BigInt representation of the input Uint25
+ * @example
+ * ```typescript
+ * const uint256Value: Uint256 = "0x2fd23d9182193775423497fc0c472e156c57c69e4089a1967fb288a2d84e914";
+ * const result = uint256ToBN(uint256Value);
+ * // result = 28139218908760698114965631183142736658067655190345294133754686837148899816404n
+ * ```
  */
 export function uint256ToBN(uint256: Uint256) {
```

Comment:
Hm, Sir how is "0x2fd23d9182193775423497fc0c472e156c57c69e4089a1967fb288a2d84e914" Uint256,
Grr i need to inspect now in details all this PR's

Reply:
Applogies for this error. The Hex-decimal string has now be converted a `CairoUint256` object. Such mistake won't occur again going forward.